proposals/lip-0051.md

```diff
@@ -2625,7 +2625,19 @@ def beforeCrossChainMessageForwarding(trs: Transaction, ccm: CCM, ccmFailed: boo
 
 ### Endpoints for Off-Chain Services
 
-This section specifies the non-trivial or recommended endpoints of the Token module and does not include all endpoints. TBA
+This section specifies the non-trivial or recommended endpoints of the Token module and does not include all endpoints.
+
+#### getBalances
+
+This function returns available and locked balances for all tokens associated with a given address.
+
+#### getSupportedTokens
+
+This function returns token IDs for both native and non-native tokens supported by the chain.
+
+#### getEscrowedAmounts
+
+This function returns the escrowed amounts and escrowed chain IDs for all escrowed tokens.
```

Comment:
Isn't this essentially outputing the whole escrow substore? If yes, I would also consider it trivial.

Reply:
Removed.